### PR TITLE
Field inheritance explanation

### PR DESCRIPTION
**Pull request by MeinMein**

### CODE REVIEW

File: class.dd [hunk 1]
```diff
@@ -113,13 +113,14 @@ $(H3 Fields)
 	the base class followed by a dot:)
 
 ---
-class A { int a; }
+class A { int a; int a2}
```
Comment (aG0aep6G) [inline class.dd]: Should be `int a2;` (missing semicolon).

**Comment by 9rnsr** [comment]
There is corresponding forum discussions or others?

**Comment by CyberShadow** [comment]
I think the example would be improved in a better way by renaming the `int a` field to something else, to avoid confusion with the `class A` type.

**Comment by JackStouffer** [comment]
ping @MeinMein any chance of updating this?

**Comment by MeinMein** [comment]
@9rnsr None that I know of. This is it  :sweat_smile:
@CyberShadow I think the field naming was chosen to show the origin of the field. For the purpose of my addition ```a2``` works in showing that you can access fields defined only in the parent class.

**Comment by andralex** [comment]
OK as is, everyone feel free to build further on this.

**Pull request merged**